src/pages/iou/request/step/IOURequestStepConfirmation.tsx

```diff
@@ -538,7 +539,9 @@ function IOURequestStepConfirmation({
                     <HeaderWithBackButton
                         title={headerTitle}
                         onBackButtonPress={navigateBack}
-                        shouldShowThreeDotsButton={requestType === CONST.IOU.REQUEST_TYPE.MANUAL && (iouType === CONST.IOU.TYPE.SUBMIT || iouType === CONST.IOU.TYPE.TRACK)}
+                        shouldShowThreeDotsButton={
+                            requestType === CONST.IOU.REQUEST_TYPE.MANUAL && (iouType === CONST.IOU.TYPE.SUBMIT || iouType === CONST.IOU.TYPE.TRACK) && !isMovingTransactionFromTrackExpense
```

Comment:
I tried replace the tracked expense receipt and BE will revert it back to the original receipt, so I hide the 3-dot button when moving tracked expense.

Reply:
This one
<img width="402" alt="image" src="https://github.com/Expensify/App/assets/50919443/6ca06373-5637-4862-835e-21f56edce17d">

In `main`, the 3-dot button doesn't show because the requestType is SCAN, but because we fix it here, the requestType is now manual.

Reply:
https://github.com/Expensify/App/assets/50919443/4e331b39-1eed-4b40-b3fe-e0aee4fcf354